web/src/main/java/com/futurewei/alcor/web/entity/route/FixedIp.java

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.web.entity.route;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.futurewei.alcor.web.entity.port.PortEntity;
+
+public class FixedIp {
```

Comment:
FixedIp is also defined in PortEntity https://github.com/futurewei-cloud/alcor/blob/d8233164736342bab40ffba06705aa197c1e5a2e/web/src/main/java/com/futurewei/alcor/web/entity/port/PortEntity.java#L251

Could you check if we could reuse it?

Reply:
Yes ~ I see the FixedIp is the internal class in PortEntity, could we move the entity to the general directory? Because some other class also need it